model_analyzer/triton/client/client.py

```diff
@@ -105,6 +105,10 @@ def load_model(self, model_name, variant_name="", config_str=None):
             return None
         except Exception as e:
             logger.info(f"Model {variant_name} load failed: {e}")
+            if hasattr(e, "_msg") and "polling is enabled" in e._msg:
```

Comment:
Instead of accessing `self._msg`, I think we can use `message` method from `InferenceServerException` class:

```python
if "polling is enabled" in e.message():
    ...
```

https://github.com/triton-inference-server/client/blob/d257c0e5c3de6e15d6ef289ff2b96cecd0a69b5f/src/python/library/tritonclient/utils/__init__.py#L71

Reply:
Done